### PR TITLE
Release v1.0.10-1

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Zcash 1.0.10
+Zcash 1.0.10-1
 =============
```
Comment (daira) [inline README.md]: This is syntactically incorrect Markdown because the line of = doesn't reach to the end of the title.

Comment (ebfull) [inline README.md]: Github doesn't particularly care about this AFAIK.

File: contrib/debian/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+zcash (1.0.10+1) stable; urgency=medium
```
Comment (daira) [inline contrib/debian/changelog]: Should this be urgency=high?

**Comment by ebfull** [comment]
ACK

File: doc/man/zcashd.1 [hunk 3]
```diff
@@ -79,7 +79,7 @@ mempool will accept (default: 0 = no limit applied)
 .HP
 \fB\-par=\fR<n>
 .IP
-Set the number of script verification threads (\fB\-2\fR to 16, 0 = auto, <0 =
+Set the number of script verification threads (\fB\-4\fR to 16, 0 = auto, <0 =
```
Comment (nathan-at-least) [inline doc/man/zcashd.1]: What is this change? Is this a false-positive replacement by `make-release.py`?

File: contrib/debian/changelog (hunk 2 above)
Comment (nathan-at-least) [inline contrib/debian/changelog]: I don't think so. I don't really know the impact of the field. From [Footnote 42 linked from here](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Urgency): 

> The urgency affects how quickly a package will be considered for inclusion into the testing distribution and gives an indication of the importance of any fixes included in the upload.

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**